trulens_eval/trulens_eval/feedback/feedback.py

```diff
@@ -860,13 +889,24 @@ def run(
             cost = mod_base_schema.Cost()
             multi_result = None
 
+            # Keep track of evaluations that were skipped due to raising SkipEval.
+            skipped_exceptions = []
+
             for ins in input_combinations:
                 try:
                     result_and_meta, part_cost = mod_base_endpoint.Endpoint.track_all_costs_tally(
                         self.imp, **ins
                     )
 
                     cost += part_cost
+
+                except SkipEval as e:
+                    e.feedback = self
+                    e.ins = ins
+                    skipped_exceptions.append(e)
```

Comment:
any reason we track every skipped exception instance w/ metadata as opposed to (for example) a count?

Reply:
felt simpler to write and more future proof if we want to do more with those exceptions